### ISSUE TITLE
Any non-tag footer entries will result in versionist failing to find the tags.

### ISSUE THREAD

**Issue by hedss** [resin-io/versionist#75]
Currently, the following works with the default config:
```
My new change.

Change-Type: patch
My-Unknown-Tag: dog
```

`versionist` will pick up the semver value and use this in versioning up.

However, the following does not work:
```
My new change.

Change-Type: patch
This is the final line of the footer.
```

My current assumption is that if `versionist` doesn't see only a set of key/value tags in a footer, it'll decide the footer is a body.
We don't stringently define the footer as only key/value pairs in the commit/PR doc, so we either need to or, I think more sensibly, allow anything in the footer as long as the relevant tags exist there.

**Comment by lurch** [comment]
But OTOH if you're allowing non-tagged entries in the footer, does it even make sense to talk about the 'footer' at all? Could the `Change-Type:` line end up being at any location in the commit message?

**Comment by hedss** [comment]
No, the main reason for allowing this is to use the 'Signed off' feature of git. I would not expect generic messages in the footer. Having the tags in the footer mean we know exactly where to look for them.
I should probably have been more explicit, but I can foresee other tools adding info messages to the bottom of a commit. :-)
